cibuildwheel/macos.py

```diff
@@ -34,21 +38,84 @@ def call(args: Sequence[PathOrStr], env: Optional[Dict[str, str]] = None, cwd: O
     return subprocess.check_call(args, env=env, cwd=cwd, shell=shell)
 
 
+def get_macos_version() -> Tuple[int, int]:
+    '''
+    Returns the macOS major/minor version, as a tuple, e.g. (10, 15) or (11, 0)
+
+    These tuples can be used in comparisons, e.g.
+        (10, 14) <= (11, 0) == True
+        (10, 14) <= (10, 16) == True
+        (11, 2) <= (11, 0) != True
+    '''
+    version_str, _, _ = platform.mac_ver()
+    version = tuple(map(int, version_str.split(".")[:2]))
+    return cast(Tuple[int, int], version)
+
+
+def get_xcode_version() -> Tuple[int, int]:
```

Comment:
This function is not called anymore and can be removed.

Reply:
Good catch 👍